js/widgets/musickeyboard.js

```diff
@@ -11,40 +11,29 @@
 // License along with this library; if not, write to the Free Software
 // Foundation, 51 Franklin Street, Suite 500 Boston, MA 02110-1335 USA
 
-/*global logo,blocks, _, platformColor, docById, MATRIXSOLFEHEIGHT, toFraction, Singer,
-   SOLFEGECONVERSIONTABLE, slicePath, wheelnav, DEFAULTVOICE, PITCHES, PITCHES2,
-   MATRIXSOLFEWIDTH, i18nSolfege, getNote, FIXEDSOLFEGE1, NOTESSHARP, NOTESFLAT,
-   last, SHARP, FLAT, PREVIEWVOLUME, noteToFrequency, SOLFEGENAMES, PITCHES3,
-   EIGHTHNOTEWIDTH, convertFromSolfege, FIXEDSOLFEGE*/
-
+/* global docById,platformColor,FIXEDSOLFEGE,FIXEDSOLFEGE1,SHARP,FLAT,last,Singer,_,noteToFrequency,
+   EIGHTHNOTEWIDTH,MATRIXSOLFEHEIGHT,i18nSolfege,MATRIXSOLFEWIDTH,toFraction,wheelnav,slicePath,blocks,getNote,PREVIEWVOLUME,
+   DEFAULTVOICE,PITCHES3,SOLFEGENAMES,SOLFEGECONVERSIONTABLE,NOTESSHARP,NOTESFLAT,PITCHES,PITCHES2,convertFromSolfege,logo */
 /*
-     Globals location
-     - lib/wheelnav
-         slicePath, wheelnav
-     
-     - js/utils/musicutils.js
-         EIGHTHNOTEWIDTH, noteToFrequency, PITCHES, PITCHES2, NOTESSHARP, NOTESFLAT,
-         FLAT, SHARP, MATRIXSOLFEHEIGHT, toFraction, SOLFEGECONVERSIONTABLE, DEFAULTVOICE,
-         MATRIXSOLFEWIDTH, i18nSolfege, FIXEDSOLFEGE1, SOLFEGENAMES, PITCHES3,
-         getNote, convertFromSolfege, FIXEDSOLFEGE
-     
-     - js/utils/utils.js
-         _, last, docById
-     
-     - js/turtle-singer.js
-         Singer
-     
-     - js/utils/platformstyle.js
-         platformColorcl
-    
-     - js/activity.js
+   Global Locations
+    - lib/wheelnav
+        slicePath, wheelnav
+    - js/utils/utils.js
+        _,docById,last
+    - js/turtle-singer.js
+        Singer
+    - js/utils/musicutils.js
+        noteToFrequency,getNote,FIXEDSOLFEGE,FIXEDSOLFEGE1,SHARP,FLAT,EIGHTHNOTEWIDTH,MATRIXSOLFEHEIGHT,i18nSolfege,MATRIXSOLFEWIDTH,
+        toFraction,DEFAULTVOICE,PITCHES,PITCHES2,PITCHES3,SOLFEGENAMES,SOLFEGECONVERSIONTABLE,NOTESSHARP,
+        NOTESFLAT,convertFromSolfege
+    - js/utils/platformstyle.js
+        platformColor
+    - js/activity.js
```

Comment:
Same for locations here.

Reply:
Actually, in PR #2849, it was committed as `platformColorcl` under the `js/utils/platformstyle.js`.
So, I just added the global locations from my closed PR #2841.
But yeah, my bad. This will not happen again surely.

Reply:
So just correct that ? do a quick rebase (:

Reply:
Yeah....will not happen again. SURELY.
My bad 😔